refactor for mutliple movie support

panatale1 opened the pull request:
Refactors code for looping
Adds support for both GB1 and GB2 wing light patterns
Adds setup screens to prompt for which pattern to use and whether to play sounds

panatale1 committed:
refactor for mutliple movie support

Refactors code for looping
Adds support for both GB1 and GB2 wing light patterns
Adds setup screens to prompt for which pattern to use and whether to play sounds

CountDeMonet commented on Arduino/PKE_METER/PKE_METER.ino:
```diff
@@ -38,16 +56,18 @@ void setup() {
   pinMode(BUZZER, OUTPUT);
   pinMode(BUTTON1, INPUT);
   pinMode(BUTTON2, INPUT);
+  pinMode(2, OUTPUT);
```
if you could add a comment here for the pin 2 output and put a comment on why they are there I'll accept the pr.

panatale1 replied:
Done. I moved the two to be closer to each other, as well, and added comments explaining why they're there. They don't ever get touched outside of the setup.

panatale1 committed:
Adds comments for mystery pin

D2 is acting as VCC for the OLED screen in my current setup running without the buck converter. This adds comments for lines I failed to comment earlier.
